### DISCUSSION TITLE
Super slow speeds?

### DISCUSSION THREAD

**Discussion by RumiAllbert** [Vinzent03/obsidian-git#177] [Problems]
Anyone else gets incredibly slow speed in obsidian after installing this plugin? I love this plugin, but it seems that every time I stage, commit, and push obsidian freezes for a few seconds, if not minutes. It's very frustrating. Anyone got a fix for this?

**Comment by Vinzent03** [comment]
I'm 90% sure you're on MacOS, right? See #69. I don't have a mac so I'm not able to test changes and I'm not familiar with web workers. So I'm not currently working on it.

**Reply by RumiAllbert** [reply]
Yes, running Monterey on MacOS. Thanks for linking the other discussion. I will patiently wait for any updates. Thank you
